### ISSUE TITLE
Text selection and inline issues

### ISSUE THREAD

**Issue by equisbacon** [asjqkkkk/markdown_widget#43]
When I use **bold** text and normal text in the same line it doesn't allow me select both the texts, I have to do one each time, and if the second text is bigger than the remaining line space it breaks to the next line, those issues could be solved by using the `extended_text` (https://pub.dev/packages/extended_text)

**Comment by asjqkkkk** [comment]
Yes, this is beacuase `SelectableText.rich` only supports `TextSpan`， but some tag widgets are made up of `WidgetSpan` such as `img`

**Comment by CiriousJoker** [comment]
So can this be fixed? I'm running into the same issue with this:

```markdown
Some long text that stretches across
multiple lines [some link](example)
```

becomes:


Some long text that stretches across
multiple lines
 [some link](example)

**Comment by asjqkkkk** [comment]
This issue has been solved with `markdown_widget 2.0.0`
